packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -68,15 +66,28 @@ class TransactionService {
     this.currentTransaction = value
   }
 
-  startCounter(transaction) {
-    transaction.browserResponsivenessCounter = 0
-    const id = setInterval(function() {
-      if (transaction.ended) {
-        window.clearInterval(id)
-      } else {
-        transaction.browserResponsivenessCounter++
+  ensureRespInterval() {
+    let currentTr = this.getCurrentTransaction()
+
+    const clearRespInterval = () => {
+      window.clearInterval(this.respIntervalId)
```

Comment:
nit: use window on both `setInterval` and `clearInterval` or can be removed from both places.